roles/calico/defaults/main.yaml

```diff
@@ -2,9 +2,28 @@
 cni_conf_dir: "/etc/cni/net.d/"
 cni_bin_dir: "/opt/cni/bin/"
 
-calico_url_policy_controller: "quay.io/calico/kube-controllers:v3.4.0"
-calico_node_image: "quay.io/calico/node:v3.4.0"
-calico_cni_image: "quay.io/calico/cni:v3.4.0"
+calico_url_policy_controller: "quay.io/calico/kube-controllers:v3.5.0-beta.0"
+calico_node_image: "quay.io/calico/node:v3.5.0-beta.0"
+calico_cni_image: "quay.io/calico/cni:v3.5.0-beta.0"
+calicoctl_image: "quay.io/calico/ctl:v3.5.0-beta.0"
```

Comment:
Do we want to move all the default images to `v3.5.0-beta` now or wait until it releases?

Reply:
Good point, we should wait until v3.5.0 is released which will be the end of this week.

Reply:
I'll put this on hold so that it won't get accidentally merged with beta images